mitodl/ocw-course-hugo-theme
# quote shortcode

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-course-hugo-theme/issues/10

#### What's this PR do?
This PR adds the `quote.html` shortcode.  It takes 2 positional parameters, the first being the quote and the second being the author, like so:

```
{{< quote "It’s such a blast to teach [this course]." "—Dr. George Kocur" >}}
```

#### How should this be manually tested?
 - Clone this PR's branch of `ocw-course-hugo-theme` and run `npm start`
 - Convert `6-034-artificial-intelligence-fall-2010` using [this](https://github.com/mitodl/ocw-to-hugo/pull/170) branch of `ocw-to-hugo`
 - Clone `ocw-course-hugo-starter` and copy the contents of `6-034-artificial-intelligence-fall-2010.json` from the `ocw-to-hugo` output into `data/course.json` 
 - Replace `ocw-course-hugo-starter`'s `go.mod` contents with the following, replacing my path to `ocw-course-hugo-theme` with yours:
```
module github.com/mitodl/ocw-course-hugo-starter

go 1.13

replace github.com/mitodl/ocw-course-hugo-theme => /home/gumaerc/Code/ocw-course-hugo-theme

require github.com/mitodl/ocw-course-hugo-theme v1.0.1 // indirect

```
 - Run `hugo server --contentDir ~/Code/ocw-to-hugo/private/output/content/courses/6-034-artificial-intelligence-fall-2010/`, replacing  my path to `ocw-to-hugo` with yours
 - Browse to http://localhost:1313/sections/instructor-insights/teaching-heuristics/ and ensure that the instructor quotes are rendering like the screenshots below

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/105546430-6191e580-5ccb-11eb-960d-32bf2b8a7c61.png)
![image](https://user-images.githubusercontent.com/12089658/105546505-78383c80-5ccb-11eb-8327-7d85d6e76858.png)
